templates/vm.tftpl

```diff
@@ -0,0 +1,136 @@
+%{ if vm_win_hostname != "" ~}module "${vm_win_hostname}" {
+  source              = "qbeyond/windows-vm/azurerm"
+  version             = "5.0.1"
```

Comment:
hardcoded versions are no good idea. More dependendencies. Please add a free text like "use latest version. Same for UNIX